src/functions/locale.rs

```diff
@@ -20,29 +23,26 @@ pub fn set_timezone(timezone: &str) {
 }
 
 pub fn set_locale(locale: String) {
-    files_eval(
```

Comment:
please revert this, it's added on purpose to make sure that at least one valid locale is generated.

Reply:
sorry, i reverted it